vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -38,41 +38,10 @@ packages:
      - six
      - docker-py
   centos:
-    rhel7:
-        packages:
-         - curl
-         - dnsmasq
-         - NetworkManager
-         - nmap
-         - OVMF
-         - patch
-         - psmisc
-         - vim-enhanced
-         - wget
-         - ansible
-         - bind-utils
-         - jq
-         - libguestfs-tools
-         - redhat-lsb-core
-         - unzip
-         - genisoimage
-         - qemu-kvm
-         - libvirt
-         - libvirt-devel
-         - libvirt-daemon-kvm
-         - libvirt-python
-         - libguestfs-tools
-         - python-lxml
-         - polkit-pkla-compat
-         - python-netaddr
-         - virt-install
-         - python-six
-         - httpd-tools
```

Comment:
some packages were python only, so we don't need to support them anymore I guess, because in common its only python3?

Reply:
I'm not quite sure. Because, python2 was shipped with Centos7. And we still have conditional checks for Centos7.
And I think it was needed for some other software ... but I again I'm  not 100% sure who is using them. We could of course deprecate them but maybe after we depreciate the support for Centos7 ?.

Reply:
This can be answered by the community specially the Centos users. Meanwhile we can atleast keep python packages since you are completely removing them. I don't know whats the best way. @russellb @hardys do you guyz have any insight here?

Reply:
Actually that's my bad. I didn't realize that I added 3 in front of python :). I would rather keep as it it now
```
         - libvirt-python
         - python-lxml
         - python-netaddr
         - python-six
```

Reply:
Would ^ be okay for now @kashifest ? and if someone from the community with more insights on Centos confirms, then maybe we can do it with python3 ?

Reply:
I'm not aware of anyone still testing with centos/RHEL7 so it may be best to de-duplicate this by deprecating centos7 support then removing the package list?

For now I don't think we can de-duplicate, because as you mention the python packages are named differently due to the change to python3 in centos8.